Formula/netdata.rb

```diff
@@ -0,0 +1,61 @@
+class Netdata < Formula
+  desc "Distributed real-time performance and health monitoring"
+  homepage "https://my-netdata.io/"
+  url "https://github.com/firehol/netdata/releases/download/v1.9.0/netdata-1.9.0.tar.bz2"
+  sha256 "542b4799aed1ee03b0a0dbd00fae988a788622c431382e6429362fbbb4f0f017"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ossp-uuid"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+
+    (etc/"netdata").install "system/netdata.conf"
+  end
+
+  def post_install
+    config = etc/"netdata/netdata.conf"
+    inreplace config do |s|
+      s.gsub!(/web files owner = .*/, "web files owner = #{ENV["USER"]}")
+      s.gsub!(/web files group = .*/, "web files group = #{Etc.getgrgid(prefix.stat.gid).name}")
+    end
+  end
+
+  plist_options :manual => "#{HOMEBREW_PREFIX}/sbin/netdata -D"
+
+  def plist; <<~EOS
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <!-- key>OnDemand</key>
```

Comment:
Why is this commented out? Uncomment or remove, please.

Reply:
Dropped. Probably reminiscence from example plist.